modules/ztp-site-cleanup.adoc

```diff
@@ -17,9 +17,20 @@ You can remove a managed site and the associated installation and configuration
 .Procedure
 
 . Remove a site and the associated CRs by removing the associated `SiteConfig` and `PolicyGenerator` or `PolicyGentemplate` files from the `kustomization.yaml` file.
+
+. Add the following `syncOptions` field to your `SiteConfig` application.
```

Comment:
Should be a colon I think, but I'm not going to hold up a merge because of this.

```suggestion
. Add the following `syncOptions` field to your `SiteConfig` application:
```